guides/v2.1/architecture/extensibility.md

```diff
@@ -32,6 +32,22 @@ Adherence to standard best practices for [PHP](https://glossary.magento.com/php)
 
 Magento has well-defined upgrade and versioning strategies that can help you avoid any problems with software component dependencies. Add modules after confirming that the [module](https://glossary.magento.com/module) version is compatible with the Magento Framework version.
 
+## Extension Points
+
+While each module can introduce its own extensibility points guided by the above principles, there is a number of extension points provided by Magento Framework or Magento core modules that are not specific to a certain feature.
+Below is a list of such generic extension points:
+* PHP interfaces marked with `@api`
+* Plugins
```

Comment:
Why plugins are extensions points?

Reply:
Removed those changes to not complicate this PR. Will deliver this separately.